bamboo/bootstrap-publish.sh

```diff
@@ -17,3 +17,4 @@ apt-get install -y jq rsync zip
 echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
 git config --global user.name "Bamboo CI"
+git config --global user.email "cumulus.bot@gmail.com"
```

Comment:
in master, publish is failing because no git configured user email can be detected. I'm not 100% sure why this is happening now an don't before, but the obvious culprit is the switch to elastic agents, after which this started happening